### ISSUE TITLE
Doesn't play nice with empathy

### ISSUE THREAD

**Issue by ara4n** [matrix-org/purple-matrix#1]
[viccuad]
mmh, trying purple-matrix, can't join or see rooms (yet I'm correctly connected, or so it says)

[Matthew (@matthew:matrix.org)]
hmmm
with what client?
viccuad: purple-matrix is pretty basic but works well enough for me with pidgin on OSX
(although hasn't been tested in a month or two)

[viccuad]
@matthew:matrix.org: empathy, on Debian Testing

[Matthew (@matthew:matrix.org)]
interesting. so that's going via the telepathy-to-libpurple bridge thingy?
we've only tested it against pidgin, so wouldn't be surprised if we've buggered something up

[viccuad]
uuh, I have no idea. I suppose

[Matthew (@matthew:matrix.org)]
bug reports / logs / patches welcome

[viccuad]
FWIW, a libpurple plugin for telegram works fine with empathy

[Matthew (@matthew:matrix.org)]
Matthew (@matthew:matrix.org)
we'll take a look

**Comment by viccuad** [comment]
I can connect, yet in Empathy the account appears greyed out in the extensible for selecting rooms to join.

here's the output for matrix on empathy-debugger, with a clean state and enabling the Matrix account, where one can see it connects and syncs rooms: https://gist.github.com/viccuad/de8fc165bd31cc32725f

**Comment by richvdh** [comment]
Libpurple distinguishes two types of conversations: IMs, which can only ever be between you and another contact; and chats, which can have multiple participants which come and go.

Matrix rooms are implemented as chats by purple-matrix, and telepathy-haze (which implements the bridge between empathy and libpurple) only supports IMs (see http://cgit.freedesktop.org/telepathy/telepathy-haze/tree/src/im-channel-factory.c#n432).

In short, this would need support from telepathy-haze to work.

**Comment by renyuneyun** [comment]
As a portal to the telepathy bug tracker for this bug (feature request):
https://bugs.freedesktop.org/show_bug.cgi?id=13127
